app-tasks/rf/setup.py

```diff
@@ -64,7 +64,7 @@ def read(*names, **kwargs):
         # eg: 'keyword1', 'keyword2', 'keyword3',
     ],
     install_requires=[
-        # eg: 'aspectlib==1.1.1', 'six>=1.7',
+        read('requirements.txt')
```

Comment:
Can't really take credit for it -- pretty sure the function was here because of the cookiecutter template used originally. When moving things around for this PR I was trying to figure out how to keep one copy of everything and saw the `read` function above already.